website-next/components/projects/projects-page-component.tsx

```diff
@@ -529,13 +529,15 @@ export class ProjectsPage extends React.Component<EmptyProps, ProjectsState> {
                 <label>Sort</label>
                 <SortButton
                   selected={this.state.sortMode === 'date'}
+                  // eslint-disable-next-line react/jsx-no-bind
```

Comment:
I think these are remnants and can be removed

Reply:
actually all this code is inside an old React class (😳), and now we're linting it properly so it throws this error. I'm not sure if we need to fix it though